hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -910,75 +869,34 @@ public Object putFromLoad(Data key, Object value) {
 
     @Override
     public Object putFromLoad(Data key, Object value, long ttl) {
-        final long now = getNow();
-
+        long now = getNow();
         if (shouldEvict(now)) {
             return null;
         }
-        markRecordStoreExpirable(ttl);
-
-        Record record = getRecordOrNull(key, now, false);
-        Object oldValue = null;
-        if (record == null) {
-            value = mapServiceContext.interceptPut(name, null, value);
-            record = createRecord(key, value, ttl, now);
-            records.put(key, record);
-            updateSizeEstimator(calculateRecordHeapCost(record));
-        } else {
-            oldValue = record.getValue();
-            value = mapServiceContext.interceptPut(name, record.getValue(), value);
-            updateSizeEstimator(-calculateRecordHeapCost(record));
-            updateRecord(record, value, now);
-            updateSizeEstimator(calculateRecordHeapCost(record));
-            updateExpiryTime(record, ttl, mapContainer.getMapConfig());
-        }
-        saveIndex(record);
-
-        return oldValue;
+        return putInternal(key, value, ttl, false, false, false, false);
     }
 
     @Override
     public boolean tryPut(Data key, Object value, long ttl) {
-        putInternal(key, value, ttl, false);
+        checkIfLoaded();
+
+        putInternal(key, value, ttl, false, false, false, false);
```

Comment:
Can you introduce some constants for these variables? Because it doesn't make much sense.. false false false false..